_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -16,74 +16,72 @@ input_aspects:
+  previous_authors:
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+This rule applies to any [HTML element][] with one or more [visible][] [text node][] child, when all the following are true for the `letter-spacing` property of the element:
 
-## Expectation
+- the [specified][] value is [declared][] in a `style` attribute; and
+- the [computed][] value is [important][].
 
-For each test target, at least one of the following is true:
+## Expectation
 
-- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
-- **wide enough**: the [computed][] value of its [letter-spacing][] property is at least 0.12 times the [computed][] value of its [font-size][] property; or
-- **cascade**: the [cascaded][] value of its [letter-spacing][] property is not a value [declared][] in its `style` attribute.
+For each test target, the [computed][] value of its `letter-spacing` property is at least 0.12 times the [computed][] value of its `font-size` property.
 
 ## Assumptions
 
-- There is no mechanism available on the page to adjust [letter-spacing][]. If there is such a mechanism, it is possible to fail this rule while [Success Criterion 1.4.12 Text Spacing][sc1412] is still satisfied.
+- There is no mechanism available on the page to adjust `letter-spacing`. If there is such a mechanism, it is possible to fail this rule while [Success Criterion 1.4.12 Text Spacing][sc1412] is still satisfied.
+
+- The font size is constant for all text in the element. If `font-size` changes (e.g., through use of the `::first-line` pseudo-element) then the required letter spacing would also change throughout the element. This is untested by the current rule.
 
 - This rule assumes that WCAG's meaning for the "Letter spacing style property" is the value of the CSS `letter-spacing` property rather than the actual space between letters. The value of the CSS property is _added_ to whichever spacing already exist (for example, due to kerning). Thus, the actual space between letters can be larger than the value of the `letter-spacing` property. If [Success Criterion 1.4.12 Text Spacing][sc1412] is concerned by the actual space between letters, then this rule may fail (with the `letter-spacing` property being too small) while the Success Criterion is still satisfied (with the actual space being enough).
 
 - This rule assumes that when inter-letters space is changed because of justification, the `letter-spacing` property is not changed. Therefore, whether a text is justified or not doesn't change the result of this rule. Note that justifying text is a failure of [Success Criterion 1.4.8 Visual Presentation][sc148].
 
+- The target text node expresses something in a human language written in a script that that uses the `letter-spacing` property.
```

Comment:
I'm not sure I understand what you mean with this assumption... the text expresses something written in a script?

Reply:
"(text expresses something) in a (human language written in a script that use letter spacing)".

For (as far as I know) written Arabic (and other languages), the letters are connected together and `letter-spacing` does not apply. ([MDN warning](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing#internationalization_concerns))
This matches WCAG's exception:
> Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.

So, essentially trying to say "if the target is text in Arabic, then `letter-spacing: 0 !important` is OK, but the rule cannot check that". (note that this is also a problem with the existing rules, so we can as well take this assumption out and deal with the problem in a separate PR…)

Reply:
Got it. It was the "script" that threw me off... I was in another context!